cosmos-gui/README.md

```diff
@@ -238,6 +241,83 @@ Next coming versions of the GUI will allow the users to explore their HDFS space
 
 [Top](#top)
 
+##<a name="administration"></a>Administration
+
+Two are the sources of data, the logs and the database, useful for an admistrator of cosmos-gui.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/8f80753e03dc1aa379cde064950e6e5de58d8eda